source/_patterns/04-pages/grid-examples/three-column/article-3-col.twig

```diff
@@ -0,0 +1,47 @@
+{% set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['grid', 'grid--article']) }) -%}
+
+<div {%- include 'atoms-attributes' %}>
+
+    <div class="site-header-wrapper grid grid-item-span-all">
+        <div class="site-header">Site header goes here</div>
+    </div>
+
+    <main class="grid grid-item-span-all">
```

Comment:
Because only the immediate children of a CSS grid container are grid items, we need make `<main>` a grid container as well to control the layout of its children.